packages/noports_core/lib/src/common/validation_utils.dart

```diff
@@ -1,4 +1,5 @@
 import 'dart:convert';
+import 'dart:io';
```

Comment:
I was very meticulous to wrap all io operations into src/common/io_types.dart and src/common/file_system_utils.dart so that we can mock it in the unit tests and ensure that we have virtualized operations for the pure dart client.

Reply:
was an unnecessary import, fixed